### PR TITLE
DOC: document model parameters of SDCA

### PR DESCRIPTION
**Pull request by fabianp**

### CODE REVIEW

File: lightning/impl/sdca.py [hunk 1]
```diff
@@ -123,15 +146,34 @@ class SDCARegressor(BaseRegressor, _BaseSDCA):
...
 
-    def __init__(self, alpha=1.0, l1_ratio=0, loss="squared", gamma=1.0,
+    def __init__(self, alpha=1.0, l1_ratio=0, loss="squared",
                  max_iter=100, tol=1e-3, callback=None, n_calls=None, verbose=0,
                  random_state=None):
         self.alpha = alpha
         self.l1_ratio = l1_ratio
         self.loss = loss
-        self.gamma = gamma
+        self.gamma = 1.0
```
Comment (mblondel) [inline lightning/impl/sdca.py]: Why did you change this?

Comment (fabianp) [inline lightning/impl/sdca.py]: As far as I can see, none of the loss functions accepted by the *Regressor object support the gamma parameter, making it a useless parameter.

Comment (mblondel) [inline lightning/impl/sdca.py]: Ah yes I had missed that :)
